doc-content/enterprise-only/ai/ai-credit-card-con.adoc

```diff
@@ -0,0 +1,117 @@
+
+The following formula in the DMN model uses the scores from the two decision tables to determine whether to automatically charge back the disputed amount (*Milestone 1*) or whether more investigation is needed (*Milestone 2*).
+[source]
+----
+(Cardholder Risk Rating + Dispute Risk Rating) < 5
+----
+If Joe's overall risk score is less than five, his disputed amount is charged back automatically (*Milestone 1*). If the overall score is five or greater, then his dispute is processed manually (*Milestone 2*).
+
+== Adding machine learning to {PRODUCT} digital decisioning
+Because Fortress Bank has historical data about its customers, including previous transaction and dispute history, the bank can use that data with machine learning to create predictive models that can be used in the DMN model decision tasks. This results in a more accurate assessment of the risk compared to the decision tables created by business analysts.
+
+Fortress Bank has two sets of PMML files that contain values for the DMN model input variable. One set is based on the linear regression algorithm and the other is based on the random forests algorithm.
+
+image:ai/PMML-models.png[]
+
+Linear regression is one of the most widely used algorithms in both statistics and machine learning. It uses a linear equation that combines a set of numeric input and output values. Random forests use many decision trees as inputs to create prediction models.
+
+.Adding PMML files
+Michelle imports the *dispute_risk_linear_regression* PMML file into her project. She adds the *Cardholder Risk Model* business model knowledge node to the DMN model and associates the *dispute_risk_linear_regression* PMML file with the node. {PRODUCT} analyzes the PMML file and add input parameters to the node.
+
+image:ai/function.png[]
+
+Michelle associates the *Cardholder Risk Model* node with the *Dispute Risk Rating*.
+
+image:ai/arrow.png[]
+
+Michelle then adds the *credit_card_holder_risk_linear_regression* PMML model to the projects, creates the *Dispute Risk Model* mode DMN file, creates, and associates the *credit_card_holder_risk_linear_regression* PMML file with the node. {PRODUCT} analyzes the PMML file and add input parameters to the node.
+
+The following image is Michelle's completed DMN model:
+//<image 7:56> <9:26>
+
+image:ai/DMN-PMML.png[]
+
+Michelle now returns to the *fraudDispute* BPMN model and updates the model with the PMML files that she added. She then redeploys.
+
+.Increased score precision
+After Michelle redeploys the project, Joe logs in to his Fortress Bank account and reports the same transaction as incorrect. In {CENTRAL}, Michelle navigates to the *Process Instances* window and she sees Joe's new dispute instance, `FR-00000005`.
+In the *Process Variables* tab, Michelle reviews the values of *cardHolderRiskRating* and the *disputeRiskRating*. They have changed because the model is now using the PMML files. This give a much more precise prediction of risk.
```

Comment:
```suggestion
In the *Process Variables* tab, Michelle reviews the values of *cardHolderRiskRating* and the *disputeRiskRating*. They have changed because the model is now using the PMML files. This give a much more precise prediction of risk by making use of machine learning models based on historical data. At the same time, the policy of the bank is still enforced by the decision model with DMN, that is the risk predictor being below a certain threshold, allow for this dispute to be process automatically.
```